security/cmd/istio_ca/main.go

```diff
@@ -112,6 +115,18 @@ var (
 			runCA()
 		},
 	}
+	// ServiceAccount/DNS pair for generating DNS names in certificates.
+	// TODO: move it to a configmap later when we have more services to support.
+	webhookServiceAccounts = []string{
+		"istio-sidecar-injector-service-account",
+		"istio-mixer-validator-service-account",
+		"istio-pilot-service-account",
+	}
+	webhookServiceNames = []string{
+		"istio-sidecar-injector",
+		"istio-mixer-validator",
+		"istio-pilot",
+	}
```

Comment:
Users will need to rebuild Istio CA if they want to run more than multiple webhooks scoped to different non-overlapping sets of namespaces in the same cluster. I believe @john-a-joyce and @sdake had this working in of of their clusters.

Reply:
I think eventually CA needs a config map to set those parameters. For now, what about adding them as flags?

Reply:
Exposing these through flags would be a good to start